source/content/guides/drupal-9-migration/05-drupal-9-to-pantheon.md

```diff
@@ -0,0 +1,377 @@
+---
+title: Migrate to Drupal 9 on Pantheon
+subtitle: Migrate a Drupal 9 Site to Pantheon
+description: Migrate a Composer-managed Drupal 9 Site from another platform to Pantheon.
+categories: [develop]
+cms: drupal-9
+tags: [code, launch, migrate, site, updates]
+reviewed: "2021-11-12"
+layout: guide
+showtoc: true
+permalink: docs/guides/drupal-9-migration/drupal-9-to-pantheon
+anchorid: drupal-9-migration/drupal-9-to-pantheon
+editpath: drupal-9-migration/05-drupal-9-to-pantheon.md
+---
+
+This doc shows how to migrate an existing Composer-managed Drupal 9 site from another platform to a new Drupal 9 site with [Integrated Composer](/integrated-composer) on Pantheon.
+
+## Overview
+
+In this migration you'll do the following:
+
+- Using Git, clone your existing site locally
+- Create a new Drupal 9 development site on Pantheon, and clone it locally
+
+Working locally across both the existing and new site codebases, you'll:
+
+- Copy selected portions of the existing site code into the new site code
+- Copy portions of `composer.json` from the existing site to the new site
+- Copy and commit the existing site's custom modules, themes, profiles, and other dependencies to the new site
+- Make any needed customizations to `settings.php` on the new site
+- Carry over any additional code customizations from the existing site to the new Pantheon Drupal 9 site code
+
+<Alert title="Integrated Composer and existing files" type="danger">
+
+Integrated Composer will break if `composer install` tries to modify any files that are committed in Git. To avoid this, [DO THIS THING *****]
+
+</Alert>
+
+## Important Notes
+
+- Integrated Composer sites require a [nested docroot](/nested-docroot) architecture. When copying over code from the existing site, be sure to retain the new site's nested docroot structure and `web` docroot name.
+  - Address the question of Composer packages being committed to version control, and how we need to not do that for IC
+- Partial `drupal-9/prepare-local-environment.md` says to install the terminus site clone plugin, but I'm not sure this is used?
```

Comment:
should I make that step its own partial? would we use the rest of the steps often enough without this step?

> 1. Install the [Terminus Site Clone](https://github.com/pantheon-systems/terminus-site-clone-plugin) plugin.

or should we have another partial like `prepare-local-environment-no-clone.md`?